webapp-django/crashstats/crashstats/urls.py

```diff
@@ -109,21 +109,4 @@
             url='/profile/',
             permanent=perm_legacy_redirect
         )),
-
-    # handle old-style URLs
-    url(r'^products/(?P<product>\w+)/$',
-        RedirectView.as_view(
-            url='/home/products/%(product)s',
-            permanent=perm_legacy_redirect
-        )),
-    url(r'^products/(?P<product>\w+)/versions/(?P<versions>[;\w\.()]+)/$',
-        RedirectView.as_view(
-            url='/home/products/%(product)s/versions/%(versions)s',
-            permanent=perm_legacy_redirect
-        )),
-    url('^home' + products + '/versions/$',
-        RedirectView.as_view(
-            url='/home/products/%(product)s',
-            permanent=perm_legacy_redirect
-        )),
```

Comment:
We need to keep all of these urls. The only one we're ditching are the management ones.